pkg/dao/metrics.go

```diff
@@ -178,3 +178,51 @@ func (d metricsDaoImpl) RHReposSnapshotNotCompletedInLast36HoursCount(ctx contex
 
 	return output + outputTaskless
 }
+
+func (d metricsDaoImpl) TemplatesUpdateTaskPendingTimeAverage(ctx context.Context) float64 {
+	var output float64 = -1
+	d.db.WithContext(ctx).
+		Model(&models.TaskInfo{}).
+		Select("coalesce(avg(extract(epoch from now() - queued_at)), 0.0) as pending_task_avg").
+		Where("status = ? and type = ?", config.TaskStatusPending, config.UpdateTemplateContentTask).
+		Pluck("pending_task_avg", &output)
+	return output
+}
```

Comment:
There's an existing method for getting the average pending time of all tasks, I wonder if that method can be updated to include an optional filter to return a particular task type? That would make it easy to do this in the future for a different task type.

Reply:
Changed this method according to the other comment, and we still need to have 2, one for the total average and the per type averages.